benchmark/protocol/src/main/scala/Protocol.scala

```diff
@@ -59,7 +59,7 @@ object protocol {
       .codec[MediumW]
       .codec[SmallW]
       .codec[BigW]
-      .specify1[LargeW, LargeW]("identityLarge")
-      .specify1[MediumW, MediumW]("identityMedium")
-      .specify1[BigW, BigW]("identityBig")
+      .specify1("identityLarge", Field.strict[LargeW]("in"), Type[LargeW])
```

Comment:
@timperrett you mean?
```
.specify1("foo", Field.strict[LargeW]("bar"), Type[LargeW])
.specify1[LargeW, LargeW]("foo", strict("bar"))
.specify[LargeW, LargeW]("foo", "bar".strict)
.specify1[LargeW, LargeW]("foo", "bar".strict[LargeW])
```
I don't have any strong preference. I wonder at this point if it might be worth having a macro that transforms a trait into this?